src/data/factions.ts

```diff
@@ -1,35 +1,62 @@
 import { Faction } from "@gaia-project/engine";
+export const stdCost = 'm:2c,1o ts:3/6c,2o lab:5c,3o PI:6c,4o AC:6c,6o';
+export const stdIncome = 'm:1o ts:3/4/4/5c lab:1k PI:4o,1t AC1:2k AC2:1q - round:1o,1k';
+export const stdStartIncome = '1o,1k';
 
-export const factionDesc = {
+export const factionDesc =  {
   [Faction.Terrans] : {
+    cost: stdCost,
+    income: stdIncome,
+    startIncome: stdStartIncome,
```

Comment:
Why repeat for each faction? By the way, might as well use what's in engine with faction boards to  get real data.

Reply:
I've tried to have objects with only the properties need, but was getting an error form vue, stating that not cost was in all objects. I didn't have time to investigate :(